### ISSUE TITLE
bootstrap4 supports?

### ISSUE THREAD

**Issue by flashjay** [humhub/humhub#4068]

**Comment by buddh4** [comment]
No, this would require migrating the core and all modules, but we'll need to upgrade in the next major version in combination with yii3 and fontawesome update.
